iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/match/AffOrgMatcherFactory.java

```diff
@@ -97,12 +98,17 @@ public static AffOrgMatcher createDocOrgRelationMatcher(JavaSparkContext sparkCo
      * Creates {@link AffOrgMatchVoter}s for {@link #createDocOrgRelationMatcher(JavaSparkContext, String, String, String, Float)} 
      */
     public static ImmutableList<AffOrgMatchVoter> createDocOrgRelationMatcherVoters() {
+
+        FittingOrgWordsMatchVoter fitOrgWordsMatchVoter = new FittingOrgWordsMatchVoter(ImmutableList.of(','), 3, 0.8f, 0.9f);
```

Comment:
Have you considered using spring or some other framework for parameterizing (and also wiring) beans?

Do you think there are any parameter candidates we should provide dynamically at runtime e.g. while conducting series of experiments on large input data.